usaspending_api/disaster/v2/views/overview.py

```diff
@@ -73,10 +75,12 @@ def award_outlays(self):
         ) or 0.0
 
     def total_obligations(self, funding):
-        return (
-            sum([elem["amount"] for elem in funding])
-            - latest_gtas_of_each_year_queryset().values("unobligated_balance_cpe").first()["unobligated_balance_cpe"]
-        )
+        remaining_balance = latest_gtas_of_each_year_queryset().filter(fiscal_year=current_fiscal_year()).first()
+        if remaining_balance:
+            remaining_balance = remaining_balance.unobligated_balance_cpe
+        else:
+            remaining_balance = Decimal("0.0")
+        return sum([elem["amount"] for elem in funding]) - remaining_balance
```

Comment:
Is this accurate for how this is calculated? I would have though total obligation was just a sum of obligations, rather than `budget - unobligated balance`. 

It doesn't appear to be annotated in the Sankey on DEV-5360, but in DEV-5050, we were purposely summing `obligations_incurred_total_cpe` (line 2190) I thought to get this amount directly from GTAS.

Reply:
The  closest thing I have to business logic for these values is the attachment on https://federal-spending-transparency.atlassian.net/browse/DEV-5424, which says to subtract these values.